### PR TITLE
Release conditions (Part 2)

### PR DESCRIPTION
**Pull request by ppaskaris-d2l**
[US112831](https://rally1.rallydev.com/#/detail/userstory/360464559620?fdp=true)
[US109699](https://rally1.rallydev.com/#/detail/userstory/332018308528?fdp=true)
[US109699](https://rally1.rallydev.com/#/detail/userstory/332018308528?fdp=true)
[US112859](https://rally1.rallydev.com/#/detail/userstory/360697030860?fdp=true)

This PR completes the new rc editor and removes the old one. Styled to look like the list item, but inline because it is a *draft*.

I'm stripping the HTML from the condition text until I have a solution for that.

[Screenshot](https://user-images.githubusercontent.com/30184468/75889427-eb5a6b00-5dfa-11ea-84a9-716c31fc98e6.png)

Depends on [siren-sdk stuff ➡](https://github.com/BrightspaceHypermediaComponents/siren-sdk/pull/143)

Kindly ignore all of the garbage commits following 25bc2fb, as they are my mostly futile attempts to get a green check mark.

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-conditions-editor.js [hunk 1]
```diff
@@ -49,18 +186,37 @@ class ActivityConditionsEditor extends ActivityEditorMixin(LocalizeMixin(MobxLit
...
-					text="${this.localize('btnRemoveCondition')}"
-					icon="tier1:delete"
-					data-key="${key}"
-					@click="${this._removeCondition}">
-				</d2l-button-icon>
+			<li class="d2l-list-item">
+				<span class="d2l-list-item-decoration">
+					<d2l-icon
+						icon="tier2:release-conditions"
+						style="width:30px;height:30px;">
```
Comment (kieranderson) [inline components/d2l-activity-editor/d2l-activity-conditions-editor.js]: nit: I to have all the css together, even if it's just small stuff like this.

File: components/d2l-activity-editor/lang/en.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ export default {
 	"btnCreateNew": "Create New", // create new button
 	"btnAddExisting": "Add Existing", // add existing button
 	"btnRemoveCondition": "Remove Condition", // remove condition button
+	"lblConditionsOperator": "To view this item, users must satisfy", // conditions operator label
```
Comment (kieranderson) [inline components/d2l-activity-editor/lang/en.js]: I assume the operator get's added to the end of this term

File: components/d2l-activity-editor/d2l-activity-conditions-editor.js (hunk 1 above)
Comment (ppaskaris-d2l) [reply components/d2l-activity-editor/d2l-activity-conditions-editor.js]: This is a stopgap until there's a proper tier3 icon.

File: components/d2l-activity-editor/lang/en.js (hunk 2 above)
Comment (ppaskaris-d2l) [reply components/d2l-activity-editor/lang/en.js]: It's followed by a select-list, yeah.

**Comment by ppaskaris-d2l** [comment]
Last commit is a fix for the logspam that Federation investigated re: bad fetch requests to the monolith (it was fetching something like this: https://instance.brightspace.com/d2l/api/some/path/to/activities/undefined)

**Pull request merged**